sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/ConfigurationAsyncClient.java

```diff
@@ -81,8 +81,8 @@
      * {@codesnippet com.azure.data.appconfiguration.configurationasyncclient.addConfigurationSetting#string-string-string}
      *
      * @param key The key of the configuration setting to add.
-     * @param label The label of the configuration setting to add, or optionally, null if a setting with
-     * label is desired.
+     * @param label The label of the configuration setting to add, or optionally, null if the label is not desired in
```

Comment:
Why did `null` change from label being desired to not desired? Was this something just missed before?

Reply:
It is used to be from one of Jonathon's feedback very long time ago. When I read this line again. I don't really convince myself it is intuitive and meant what I was trying to explain here. I am trying to tell the user, Label is optional but not required in the request.